src/UI/Component/Modal/Interruptive.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace ILIAS\UI\Component\Modal;
+
+/**
+ * Interface Interruptive
+ *
+ * @package ILIAS\UI\Component\Modal
+ */
+interface Interruptive extends Modal {
```

Comment:
The action button is now mandatory and the interruptive modal takes a form action where the action button will send the POST request, optionally containing the IDs of the affected items.
The label of the action button is injected as language key with the constructor: https://github.com/ILIAS-eLearning/ILIAS/pull/404/files#diff-4a17a81d57879772cca64c5300e12065R43
However, this is currently not exposed to the factory: What's the strategy here?